react/demo/app/samples/pagination/Default.route.tsx

```diff
@@ -3,13 +3,30 @@ import {useState} from 'react';
 
 const PaginationDemo = () => {
 	const [page, setPage] = useState(4);
+	const [pageAlone, setPageAlone] = useState(1);
 
 	return (
 		<>
-			<PaginationAgnosUI page={page} onPageChange={(page) => setTimeout(() => setPage(page), 0)} />
+			<h2>Default pagination:</h2>
+			<PaginationAgnosUI page={page} collectionSize={70} onPageChange={(page) => setTimeout(() => setPage(page), 0)} />
```

Comment:
Do you remember why we need the ```setTimeout``` calls ?
Trying in local without those seems to work just fine, but there's probably a good reason why we had to add them...

Reply:
yes I do, we discuss this with David-Emmanuel. We need to see how to do it in react for the output...